core-api/src/main/java/com/sequenceiq/cloudbreak/api/model/KerberosBase.java

```diff
@@ -47,9 +41,17 @@
     @RequiredKerberosField
     private Boolean tcpAllowed = false;
 
-    @NotNull
+    @NotNull(message = "Kerberos type can not be null")
     private KerberosType type;
 
+    @ApiModelProperty(StackModelDescription.KERBEROS_DOMAIN)
+    private String domain;
+
+    @Pattern(regexp = "(^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(,((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\.){3}"
```

Comment:
please write some test for this regex. not just for breaking the test if someone changes the pattern, but to provide examples through the test also.
(like in BlueprintRequestTest.java)